tasks/postgres.yml

```diff
@@ -0,0 +1,24 @@
+
+- name: Add peertube user in Postgres
+  become: yes
+  become_user: postgres
+  postgresql_user:
+    name: peertube
+    password: "{{ peertube_dbuser_password }}"
+
+- name: Add Postgres database
+  become: yes
+  become_user: postgres
```

Comment:
Use defaults variables

Reply:
[Here](https://github.com/UdelaRInterior/ansible-peertube/blob/dcd9bde9d7ec2b10ebcf07974a7a0113ab64d6c1/defaults/main.yml#L5) created variable peertube_user
[Here](https://github.com/UdelaRInterior/ansible-peertube/blob/dcd9bde9d7ec2b10ebcf07974a7a0113ab64d6c1/tasks/postgres.yml#L4), [here](https://github.com/UdelaRInterior/ansible-peertube/blob/dcd9bde9d7ec2b10ebcf07974a7a0113ab64d6c1/tasks/postgres.yml#L11), [here](https://github.com/UdelaRInterior/ansible-peertube/blob/dcd9bde9d7ec2b10ebcf07974a7a0113ab64d6c1/tasks/postgres.yml#L18), [here](https://github.com/UdelaRInterior/ansible-peertube/blob/dcd9bde9d7ec2b10ebcf07974a7a0113ab64d6c1/tasks/install_configure_peertube.yml#L4)..... it is used.
[Here](https://github.com/UdelaRInterior/ansible-peertube/blob/dcd9bde9d7ec2b10ebcf07974a7a0113ab64d6c1/defaults/main.yml#L6) created variable peertube_group
[Here](https://github.com/UdelaRInterior/ansible-peertube/blob/dcd9bde9d7ec2b10ebcf07974a7a0113ab64d6c1/tasks/install_configure_peertube.yml#L14),  [here](https://github.com/UdelaRInterior/ansible-peertube/blob/dcd9bde9d7ec2b10ebcf07974a7a0113ab64d6c1/tasks/install_configure_peertube.yml#L30)..... it is used.